src/app/modules/item/pages/item-progress/item-progress.component.ts

```diff
@@ -10,4 +12,7 @@ export class ItemProgressComponent {
 
   @Input() itemData?: ItemData;
 
+  session$ = this.sessionService.session$.pipe(delay(0));
```

Comment:
I thought it would be also used for an `item-log-view` component when watching a group.
![image](https://user-images.githubusercontent.com/31007874/100633087-e98fea80-332d-11eb-8ff9-eac968a8d5cd.png)

Reply:
Sessions (info about the current info and managed group) are something every component can access, so I don't see on what basis we would pass them by arg here and not at other places... That is not bad, but looking for consistency, I don't see reason to do that in this case.

Reply:
fixed